+p_xml: 'version="1.0" encoding="UTF-8" standalone="yes" '
+directive: DOCTYPE bugzilla SYSTEM "http://bugzilla.fieldtriptoolbox.org/page.cgi?id=bugzilla.dtd"
bugzilla:
  +@version: 4.4.1
  +@urlbase: http://bugzilla.fieldtriptoolbox.org/
  +@maintainer: r.oostenveld@donders.ru.nl
  bug:
    bug_id: "458"
    creation_ts: 2011-01-29 08:49:00 +0100
    short_desc: fixdimord cannot handle dimord voxel
    delta_ts: 2011-02-03 10:46:12 +0100
    reporter_accessible: "1"
    cclist_accessible: "1"
    classification_id: "1"
    classification: Unclassified
    product: FieldTrip
    component: core
    version: unspecified
    rep_platform: PC
    op_sys: Windows
    bug_status: CLOSED
    resolution: FIXED
    bug_file_loc:
    status_whiteboard:
    keywords:
    priority: P1
    bug_severity: major
    target_milestone: '---'
    everconfirmed: "1"
    reporter:
      +content: a.stolk8
      +@name: Arjen Stolk
    assigned_to:
      +content: r.oostenveld
      +@name: Robert Oostenveld
    cc: r.oostenveld
    comment_sort_order: oldest_to_newest
    long_desc:
      - +@isprivate: "0"
        commentid: "1318"
        comment_count: "0"
        who:
          +content: a.stolk8
          +@name: Arjen Stolk
        bug_when: 2011-01-29 08:49:17 +0100
        thetext: |-
          Since this revision

          2011-01-26 08:11  roboos

          	* [r2646] private/fixdimord.m: improved handling of dimords for source and volume data



          I get the following error:

          ??? Error using ==> fixdimord at 147
          unexpected dimord "voxel"

          Error in ==> ft_checkdata at 151
            data = fixdimord(data);

          Error in ==> ft_volumelookup at 95
            volume = ft_checkdata(volume, 'datatype', 'volume');
      - +@isprivate: "0"
        commentid: "1319"
        comment_count: "1"
        who:
          +content: a.stolk8
          +@name: Arjen Stolk
        bug_when: 2011-01-29 08:52:50 +0100
        thetext: |-
          Before that revision, a warning is given:

          unexpected dimord "voxel", dimord is removed from the data
      - +@isprivate: "0"
        commentid: "1329"
        comment_count: "2"
        who:
          +content: r.oostenveld
          +@name: Robert Oostenveld
        bug_when: 2011-01-31 18:03:48 +0100
        thetext: kun je de me input data (dwz met de voxel dimord) sturen?
      - +@isprivate: "0"
        commentid: "1338"
        comment_count: "3"
        who:
          +content: r.oostenveld
          +@name: Robert Oostenveld
        bug_when: 2011-02-01 09:10:52 +0100
        thetext: "ik heb het gefixed\n\n\nMacBook> svn commit fixdimord.m \nSending        fixdimord.m\nTransmitting file data .\nCommitted revision 2719."
      - +@isprivate: "0"
        commentid: "1372"
        comment_count: "4"
        who:
          +content: r.oostenveld
          +@name: Robert Oostenveld
        bug_when: 2011-02-02 23:43:10 +0100
        thetext: closed the bugs that were recently fixed
      - +@isprivate: "0"
        commentid: "1374"
        comment_count: "5"
        who:
          +content: a.stolk8
          +@name: Arjen Stolk
        bug_when: 2011-02-03 10:46:12 +0100
        thetext: |-
          Somehow fixdimord is now not found anymore by ft_checkdata while I can see it in the private dir.

          (In reply to comment #4)
          > closed the bugs that were recently fixed
